central/scannerdefinitions/handler/mappingUpdater.go

```diff
@@ -0,0 +1,154 @@
+package handler
+
+import (
+	"fmt"
+	"math/rand"
+	"net/http"
+	"os"
+	"time"
+
+	backoff "github.com/cenkalti/backoff/v3"
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/central/scannerdefinitions/file"
+	"github.com/stackrox/rox/pkg/concurrency"
+	"github.com/stackrox/rox/pkg/sync"
+)
+
+var (
+	_       RequestedUpdater = (*mappingUpdater)(nil)
+	randGen                  = rand.New(rand.NewSource(time.Now().UnixNano()))
+) /**/
+
+type mappingUpdater struct {
```

Comment:
We can potentially change this to v4 updater or similar since I expect CVSS bundle is also a download process.